### ISSUE TITLE
Allow guest without login

### ISSUE THREAD

**Issue by mdanetzky** [Requarks/wiki#379]
### in config.yml set 

```
public: true, 
auth:
  defaultReadAccess: true
  local:
    enabled: true
```

observe login still popping up.

**Comment by NGPixel** [comment]
Setting `public: true` will allow guest viewing. Have you restarted Wiki.js after making the change?

**Comment by mdanetzky** [comment]
Yes, I did restart it. I even upgraded the wiki.js (and lost all the pages in the process).

**Comment by NGPixel** [comment]
All content is stored in /repo folder, which is kept in sync with your remote git. The only way to loose your pages is that you delete the repo folder AND didn't configure the remote git sync.

**Comment by mdanetzky** [comment]
We know that now! We assumed, that the wiki/tools/docker-compose.yml creates all mutable volumes automatically. This is not true. We have learned our lesson the hard way and will not trust these files in the future. 
Back to main topic - public wiki is not working - so we decided to publish markdown using other means.

**Comment by NGPixel** [comment]
Nobody else has this issue so you must be doing something wrong.

- Can you share your `docker-compose.yml` and `config.yml` with private info removed?
- In the wiki, under **Account** > **Users**, is the **Guest** account listed? What are the access rights of that user?

**Comment by mdanetzky** [comment]
It is https://github.com/Requarks/wiki/blob/master/tools/docker-compose.yml
It was the default Guest user, we haven't change it.

**Comment by mbitsnbites** [comment]
I have the same (or similar?) problem.

```yaml
public: true

auth:
  defaultReadAccess: false
  local:
    enabled: true
```

When entering the top URL in the browser I get the "Forbidden" sign:

![bild](https://user-images.githubusercontent.com/1193992/48260229-f70e6b00-e41a-11e8-98b7-fa7a78d33de2.png)

**Comment by unquietwiki** [comment]
Related to issue #143

If you set "public: true" after first starting the site / container, you'll need to edit / wipe the MongoDB database to remove the guest account. It needs to be recreated with the public permission.
